### DISCUSSION TITLE
When I use some minutes this app on my M1 mac,  this app will be stuck

### DISCUSSION THREAD

**Discussion by robin-dongbin** [spatie/ray#384] [Q&A]
I don't know the reason. Every time after I output debug info, I have to restart it. Is anyone have issues like me?

**Comment by johncarter-** [comment]
I'm getting the same on Windows 10, particularly in foreachs of around ~600 items.

**Reply by robin-dongbin** [reply]
The reason is output too many items?

**Reply by johncarter-** [reply]
🤷
